meson.build

```diff
@@ -14,9 +14,9 @@ fs = import('fs')
 audit_dep = dependency('audit', required : get_option('audit'))
 if audit_dep.found()
   cc_audit_flags = '-DHAVE_AUDIT'
-  else
+else
   cc_audit_flags = []
-  endif
+endif
```

Comment:
This change is not related to the topic of this commit/PR, i.e. shouldn’t be here.